x-pack/plugins/security_solution/public/timelines/components/timeline/body/data_driven_columns/index.tsx

```diff
@@ -448,3 +448,13 @@ export const getMappedNonEcsValue = ({
   }
   return undefined;
 };
+
+export const useGetMappedNonEcsValue = ({
```

Comment:
Did we want to memoize this in `useMemo`? If not, can it just be a regular function?

Reply:
that's what i meant to do, ya.